op-node/sources/l1_client.go

```diff
@@ -151,13 +155,20 @@ func (s *L1Client) GoOrUpdatePreFetchReceipts(ctx context.Context, l1Start uint6
 						time.Sleep(3 * time.Second)
```

Comment:
It seams we can improve a little bit of performance here. The 3 seconds sleep blocks the handling of `s.preFetchReceiptsStartBlockChan` as well, so we can combine these two channel events into one : `s.preFetchReceiptsStartBlockChan` and `rateLimiter`, with a individual go routine to handle  the `rateLimiter` and its 3 seconds sleep.